scripts/models/searchModel.js

```diff
@@ -0,0 +1,38 @@
+(function(module) {
+  var searchTool = new Object();
+  searchTool.AllCars = [];
+
+  searchTool.init = function(ctx, next) {
+    next();
+  };
+
+  searchTool.getAllCars = function(callback) {
+    $.get('/vehicle/v2/makes').done(function(data) {
+      callback(data);
+      searchView.createMakeFilter();
+      searchView.handleMakeFilter();
+    });
+  };
+
+  searchTool.getCarPhoto = function(callback) {
+    $.get('/media/v2/' + searchView.currentCar.make.toLowerCase() + '/' + searchView.currentCar.model.toLowerCase() + '/' + searchView.currentCar.year + '/photos').done(function(data){
+      callback(data);
+    });
+  };
+
+  searchTool.printAllCars = function(data) {
```

Comment:
How do you print all cars by just pushing them to the array? Seems to be an unnecessary step that could be wrapped into another method.